Move pamela to console (#2118)
kyma-project/console: pamela/README.md

# PAMELA

## Overview

> This is a REST proxy for the Console

Reviewer: I'd love to know more about Pamela. Please elaborate.

> This is a REST proxy for the Console

Author: Well, we're still not sure about the functionality and shape of Pamela. That's why we use a feature branch (named _pamela_ for it). I think it would be better to write some more here once the Pamela is done.

## Prerequisites

- Node.js 12

> - NPM or Yarn

Reviewer: I'd rather mention kubeconfig here than in all the places below. What do you think?

> - NPM or Yarn

Author: A kubeconfig variable/file is definitely not a prerequisite. You can built Pamela docker image (which is one of the ways you can use it) without the kubeconfig.

## Installation

> `npm install`

Reviewer: Please elaborate.

Author: There's nothing to elaborate here, I guess. Once prerequisites are met this command is all you need to do.

> `npm install`

Reviewer: ```suggestion
Run the `npm install` command to install Pamela locally.
```

## Usage

> One of:

Reviewer: Maybe something like:
```
Run the `npm run watch` or `npm start` command to use Pamela locally. You can also use the `docker build` command to use Pamela as an image of a Kubernetes Pod.
```

> To install Pamela together with the Console, run the `npm run bootstrap` in the root directory.

Reviewer: ```suggestion
To install Pamela together with the Console, run the `npm run bootstrap` command in the root directory.
```

> One of:
> 
> - `npm run watch` (described below; KUBECONFIG env variable must be provided)
> - `npm start` ( KUBECONFIG env variable must be provided)
> - `docker build` and use it as an image of a Kubernetes pod

Reviewer: ```suggestion
Run the `npm run watch` or `npm start` command to use Pamela locally. To do so, you must provide the KUBECONFIG environment variable. You can also use the `docker build` command to use Pamela as an image of a Kubernetes Pod.
```

## Development

> Use `npm run watch` command to run Pamela in the unsafe mode (no TLS certificate) and with the _watch_ mode on.

Reviewer: ```suggestion
Use the `npm run watch` command to run Pamela in the unsafe mode without the TLS certificate and with the watch mode on.
```

> `KUBECONFIG` environment variable **must be provided**.

Reviewer: ```suggestion

```

> `KUBECONFIG` environment variable **must be provided**.

Reviewer: ```suggestion
To do so, the KUBECONFIG environment variable **must be provided**.
```
